### PR TITLE
HTS-223: Removed Option wrap from dateOfBirth in Person class, this f…

### PR DESCRIPTION
**Pull request by Jackie1988**
…ixed a eligibility check selenium test

### CODE REVIEW

**Comment by sureshhmrc** [comment]
@Jackie1988  you need to update the help-to-save backend to remove Option from there too

**Comment by sureshhmrc** [comment]
if you want to remove Option from here, you end up making many changes in `help-to-save`

**Comment by Jackie1988** [comment]
ok I have made the change  in help-to-save

**Comment by sureshhmrc** [comment]
we cannot make change in `help-to-save`  to remove Option there unles we are  100% sure that we always get DOB from real citizen-details endpoint (from qa/prod envs)

File: app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala [hunk 1]
```diff
@@ -52,8 +52,9 @@ object CitizenDetailsController extends BaseController {
     val personGen = for {
       fnameO <- Gen.forename.almostAlways
       snameO <- Gen.surname.almostAlways
```
Comment (MarcusQA) [inline app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala]: As discussed, almostAlways can return null/blank - for what we're doing at the moment, I think we need a forename and surname consistently returned.

File: app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala [hunk 2]
```diff
@@ -63,6 +64,7 @@ object CitizenDetailsController extends BaseController {
       add3 <- initAdd(2)
       postcodeO <- address.last
```
Comment (MarcusQA) [inline app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala]: As discussed, almostAlways can return null/blank - for what we're doing at the moment, I think we need a forename and surname consistently returned.

File: app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala [hunk 3]
```diff
@@ -63,6 +64,7 @@ object CitizenDetailsController extends BaseController {
...
       countryO <- const("UK").almostAlways
```
Comment (MarcusQA) [inline app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala]: As discussed, almostAlways can return null/blank - for what we're doing at the moment, I think we need address consistently returned.

**Pull request merged**